views/partials/footer.pug

```diff
@@ -1,26 +1,16 @@
 div(id="footer")
     div(class="section")
-        b PHONE NUMBER 
-        br
-        p #{footer.phoneNumber}
+        b ©2023 Mathsoc
```

Comment:
do you think we could use `new Date().getFullYear()` to make this auto-update on an ongoing basis?

Reply:
Fixed. Please check the PR again :)